.github/workflows/test.yml

```diff
@@ -25,19 +25,22 @@ jobs:
       - name: Install swaggo
         run: go install github.com/swaggo/swag/cmd/swag@v1.8.12
 
-      - name: Generate Go ebpf objects
-        run: go generate -v ./cmd/eupf
+      - name: Generate project dependencies

```

Comment:
Questionably. Whick kind of deps are updated here? Compilation of ebpf program doesn't look like deps IMHO

Reply:
The spread syntax `./cmd/...` runs go generate for all folders in cmd, that include api, where swagger files are generated.

Reply:
I'm talking about `name`

Reply:
Well then, what name do you suggest?

Reply:
Previous name looked good. May be `Compile ebpf and generate go wrappers`?